Contest_API.md

```diff
@@ -1271,12 +1271,12 @@ Properties of person objects:
 | :---------- | :-------------- | :----------
 | id          | ID              | Identifier of the person.
 | icpc\_id    | string ?        | External identifier from ICPC CMS.
-| team\_id    | ID ?            | [Team](#teams) of this person. Required iff role is `team`.
+| team\_ids   | array of ID ?   | [Team](#teams) of this person. Required iff role is `contestant` or `coach`.
```

Comment:
Maybe
```suggestion
| team\_ids   | array of ID ?   | [Team](#teams) of this person. Required to be non-empty iff role is `contestant` or `coach`.
```
? Technically an empty list is also a value but that is not what we want.

Reply:
I think we should allow empty lists. (we do for all other `array of ID`).

That said, the fact that it's nullable *and* allowed to be empty means that there are two ways to specify the same thing. Maybe it should not be nullable? (If we do that change we should do the same change in all the similar places)

Reply:
I was talking about the contestant or role persons. I think we wanted to say that those NEED a team.

Reply:
Ah, yes, of course. I agree.